uk.co.compucorp.civicrm.hrsampledata/sql/uninstall.sql

```diff
@@ -5,13 +5,6 @@ TRUNCATE TABLE civicrm_address;
 TRUNCATE TABLE civicrm_case;
 TRUNCATE TABLE civicrm_case_activity;
 TRUNCATE TABLE civicrm_case_contact;
-
-DELETE FROM civicrm_contact WHERE id !=1;
-
-DELETE FROM civicrm_email WHERE contact_id !=1;
-
-DELETE FROM civicrm_uf_match;
-
```

Comment:
Any reason why we don't need to cleanup the data in these tables anymore?

Reply:
They are cleaned here https://github.com/civicrm/civihr/blob/7eec0c7e9e9d5a2c52ad8837f19e188b01fedc99/uk.co.compucorp.civicrm.hrsampledata/CRM/HRSampleData/Upgrader.php#L46

 because there are some contacts attached to current logged in users when installing the extension via UI, it had to be done in this way.